.github/workflows/push-to-aws-s3.yaml

```diff
@@ -44,6 +44,7 @@ jobs:
           npm install crypto-browserify
           npm install stream-browserify
           npm install browserify-zlib
+          export APP_URL=https://stac.eodhp.eco-ke-staging.com
```

Comment:
Just noticed an issue with this. If we want to serve this app from multiple CloudFront distributions (which will have different URLs), how can we serve the one static deployment to S3 from multiple different URLs? @ahaywardtvuk